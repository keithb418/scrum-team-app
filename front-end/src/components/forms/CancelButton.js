import React from "react";
import { Button } from "react-bootstrap";
import { Link } from "react-router-dom";

const CancelButton = () => (
  <Link to="/">
    <Button>Cancel</Button>
  </Link>
);


export default CancelButton;
