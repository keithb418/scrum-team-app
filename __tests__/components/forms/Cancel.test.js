import React from 'react';
import { shallow } from 'enzyme';
import CancelButton from '../../../front-end/src/components/forms/CancelButton';

test('should render the Header component correctly', () => {
    const wrapper = shallow(<CancelButton />);
    expect(wrapper).toMatchSnapshot();
});