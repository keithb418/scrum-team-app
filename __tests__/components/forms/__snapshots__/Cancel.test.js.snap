// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Header component correctly 1`] = `
<Link
  replace={false}
  to="/"
>
  <Button
    active={false}
    block={false}
    bsClass="btn"
    bsStyle="default"
    disabled={false}
  >
    Cancel
  </Button>
</Link>
`;
